modules/wallet.go

```diff
@@ -364,8 +364,8 @@ type (
 		RegisterTransaction(t types.Transaction, parents []types.Transaction) TransactionBuilder
 
 		// Rescanning reports whether the wallet is currently rescanning the
-		// blockchain.
-		Rescanning() bool
+		// blockchain, and if so, the highest blockheight scanned.
+		Rescanning() (bool, types.BlockHeight)
```

Comment:
waffled on whether this should be two separate functions, or maybe `(types.BlockHeight, bool)` to match the "comma ok" idiom.